### DISCUSSION TITLE
Backup - Files Copy limitation

### DISCUSSION THREAD

**Discussion by 2020testuser** [vmware-tanzu/velero#3510] [Community support Q&A]
**What steps did you take and what happened:**
Could anyone please help me?
I'm using velero to backup my entire Kubernetes cluster.
Is there any limitation on the number of files or max files size copied by the backup command?
If so, could anyone please share the limitation details? Thanks in advance!

========
Note:
The backup command was running continuously for over 6 hours and I was unable to check the logs as the backup command was still running.

   Kubernetes version - 1.16

**Comment by dsu-igeek** [comment]
There aren't any explicit limits.  Are you using Restic to backup your data?  You can check the logs on the Restic pods to see if there are any issues.  How much data are you backing up and where is it going to?
